cfgov/jinja2/v1/_includes/organisms/email-signup-form.html

```diff
@@ -0,0 +1,44 @@
+{# ==========================================================================
+
+   email-subscribe-form.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates an email subscription form when given:
+
+   gd_code:       A GovDelivery code for a specified mailing list.
+
+   settings:      An object used to customize the markup.
+
+   settings.text: The text used used within the description markup.
```

Comment:
EDIT: Ahh `gh_code` is alphabetical. In other places we have them in the order of the parameters, but either way works.